### DISCUSSION TITLE
Can we get information about the deleted public github repository like who downloaded it ?

### DISCUSSION THREAD

**Discussion by tanmaycode2** [community/community#49331] [Code Security]
### Select Topic Area

Product Feedback

### Body

Hello guys, I had one private repository that accidently got public and once i realized its already been 8 hours. So I deleted that repository immediately but i need to check now that who else downloaded or cloned my repository when it was public for few hours.
Can somebody please answer this immediatley. As its very urgent and very important.

**Comment by tyler-holm** [comment]
You cannot see who has downloaded or cloned your repository. Repositories can be cloned by anyone, even people that don't have GitHub accounts, so there's no way to keep track of who has cloned a repo. If someone has made a fork then you would've been able to see that, but now that you've deleted the repo you won't be able to.

If you found this response useful then please mark it as the solution to your question. Thanks!
